src/ncp/ncp_base.cpp

```diff
@@ -2177,11 +2183,18 @@ ThreadError NcpBase::SetPropertyHandler_NET_ENABLED(uint8_t header, spinel_prop_
     {
         if (value == false)
         {
-            errorCode = otDisable();
+            errorCode = otDisable(mContext);
         }
         else
         {
-            errorCode = otEnable();
```

Comment:
This is related to my other otEnable comment. We cannot reinitialize mContext without reinitializing everything that has a reference to it.